ofctrl/fgraphFlow.go

```diff
@@ -1,4 +1,5 @@
-/***
+/*
```

Comment:
No need for this change?

Reply:
It is formatted by Goland automatically.